client/my-sites/checkout/checkout/notice-error-message.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+const NoticeErrorMessage = ( { message } ) => <strong>{ message }</strong>;
```

Comment:
Is this really something we need as a component? The name is a bit confusing too, I was expecting it's somehow related to our `Notice` components.

Reply:
I was working on the assumption that, if additions or changes were required to the translation variable, it would be easier via a component than updating a translation string (which has to then be re-translated/updated)  Maybe @akirk can confirm?

Reply:
Usually with something this simple we'd do it inline in the `translate()` arguments.